noisepagecontrol/control_plane/services/exploratory_worker_handler/event_handler.py

```diff
@@ -49,8 +55,57 @@ def handle_launch_exploratory_postgres_event(event):
     tuning_id = event["data"]["tuning_id"]
     event_name = event["data"]["event_name"]
 
-    launch_exploratory_postgres(tuning_id, event_name)
+    from .models import ExploratoryPGInfo
+    from .exploratory_pg_status_types import ExploratoryPGStatusType
+    from control_plane.services.tuning_manager.models import TuningInstance, TuningEvent
+
+    tuning_instance = TuningInstance.objects.get(tuning_id=tuning_id)
+    tuning_event = TuningEvent.objects.get(tuning_id=tuning_id, event_name=event_name)
+
+    replica_url = tuning_instance.replica_url
+    # Get config.snapshot to decide if exploratory postgres cluster should be
+    # launched by taking a snapshot of replica cluster. Default to false.
+    snapshot = tuning_event.config.get("snapshot", False)
```

Comment:
Instead of querying, let's add the config to the event itself; 
(in child_event_publisher L51, when we publish the child event)

Reply:
make huge sense